### PR TITLE
Javascript-and-json - Evan Hong

### PR DESCRIPTION
**Pull request by EvanS-Hong**
image

### CODE REVIEW

File: javascript-and-json/main.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  "isbn": "987812315",
+  "title": "Goodbye World",
+  "author": "Reverse Order",
+},
+{
+  "isbn": "159753654852",
+  "title": "Dlrow Eyboog",
+  "author": "Redro Dradnats",
+},
+];
```
Comment (shawnkost) [inline javascript-and-json/main.js:16]: You're missing some console logs for this array, make sure you're following all the directions. You should be logging out the value of this array, along with the `typeof`

File: javascript-and-json/main.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ var string = JSON.stringify(array);
+ console.log(string);
```
Comment (shawnkost) [inline javascript-and-json/main.js]: You should be logging the `typeof` as well

File: javascript-and-json/main.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+ var string2 = '{"numberID": "249675", "stringName": "Brandon Cho"}'
+ console.log(typeof(string2));
```
Comment (shawnkost) [inline javascript-and-json/main.js]: You should be logging out the value of this as well

File: javascript-and-json/main.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+ console.log(typeof(JSON.parse(string2)));
```
Comment (shawnkost) [inline javascript-and-json/main.js]: You should be logging out the value of this as well

**Review by shawnkost — CHANGES_REQUESTED** [review]
Just a few console logs missing here Evan!

**Comment by EvanS-Hong** [comment]
image
 My bad, misread those as putting them together in one console log.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**